content/actions/reference/environment-variables.md

```diff
@@ -39,7 +39,6 @@ We strongly recommend that actions use environment variables to access the files
 | Environment variable | Description |
 | ---------------------|------------ |
 | `CI` | Always set to `true`. |
```

Comment:
We've got another similar PR where someone is suggesting to add the Windows-specific variable rather than removing the `HOME` variable. Curious what your thoughts are there?

I can see removing the OS-specific variables, but it might be helpful as we do describe that the directory is where user data is stored.

```suggestion
| `CI` | Always set to `true`. |
| `HOME` | (On Linux and macOS runners) The path to the {% data variables.product.prodname_dotcom %} home directory used to store user data. For example, `/github/home`. |
| `HOMEPATH` | (On Windows runners) The path to the {% data variables.product.prodname_dotcom %} home directory used to store user data. For example, `c:\Users\runneradmin`. |
```

Reply:
My strongly opinion is we must not mess user home dir related and OS specific environment variable with the github specific variable. 

In fact "The path to the GitHub home directory used to store user data" has nothing to do with "home dir to store user settings"

As a result, the first step is to remove the mentioning of UNIX-specific environment variable HOME and the second step is to add something like "GITHUB_HOME" or "GITHUB_DATA" or "GITGHUB_WORKDIR" which might point to the same directory as HOME or to another.

Reply:
Personally, I agree with Sergey and @thboop mentioned the similar point https://github.com/github/docs/pull/328#issuecomment-707112891
We should not mess variables that are set by GitHub and variables are set by system. At least, because there are a ton of other system variables that we don't want to describe here.

> but it might be helpful as we do describe that the directory is where user data is stored.

Actually, it is already described `GITHUB_WORKSPACE`. 
```
The GitHub workspace directory path. The workspace directory is a copy of your repository if your workflow uses the actions/checkout action. If you don't use the actions/checkout action, the directory will be empty. For example, /home/runner/work/my-repo-name/my-repo-name.
```
I guess @dsame meant the same under `GITHUB_HOME`

Reply:
@rachmari with the GITHUB_WORKSPACE already there's no sense to mention HOME variable at all. It works for windows users but confuses *nix guys because of messing with shell HOME variable.

Reply:
Sounds good. I'll close out the other PR that is suggesting to add the `HOMEPATH` variable and I'll get this update shipped.